index.html

```diff
@@ -123,21 +123,15 @@ <h1 class="page-header" id="support">Support</h1>
         <div>
             NLog supports the following platforms:
             <ul>
-                <li> .NET 5</li>
-                <li> .NET Framework 3.5, 4, 4.5 - 4.8</li>
-                <li> .NET Framework 4 client profile</li>
-                <li>Xamarin Android</li>
-                <li>Xamarin iOs</li>
-                <li>Windows Phone 8</li>
-                <li>Silverlight 4 and 5</li>
+                <li>.NET 5 &amp; 6 (.NET Standard 2.0)</li>
+                <li>.NET Framework 3.5 - 4.8</li>
+                <li>.NET Core 1.0 - 3.1</li>
+                <li>Xamarin Android + iOS (.NET Standard)</li>
+                <li>UWP (.NET Standard)</li>
                 <li>Mono 4</li>
-                <li>ASP.NET 4 (NLog.Web package)</li>
                 <li>ASP.NET Core (NLog.Web.AspNetCore package)</li>
-                <li>.NET Core (NLog.Extensions.Logging package)</li>
-                <li>.NET Standard 1.x - NLog 4.5</li>
-                <li>.NET Standard 2.x - NLog 4.5</li>
-                <li>UWP - NLog 4.5</li>   
-              </ul>
+                <li>ASP.NET Classic (NLog.Web package)</li>

```

Comment:
Is that called "classic" nowadays?  Sounds like ASP classic ;)

Suggestion: ASP.NET or ASP.NET (non core)

Reply:
Microsoft calls it also ASP.NET 4.x: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/choose-aspnet-framework?view=aspnetcore-2.1#aspnet-4x

Reply:
Changed to ASP.NET (non core).